app/coffee/Repositories/Tags.coffee

```diff
@@ -27,6 +27,14 @@ module.exports = Tags =
 			"$addToSet": {project_ids:project_id}
 		db.tags.update(searchOps, insertOperation, callback)
 
+	addProjectToTagName: (user_id, tag_name, project_id, callback = (error) ->)->
+		searchOps =
+			tag_name: tag_name
```

Comment:
Should we done some kind of canonicalisation here, like down-casing? I'm not sure, because users may want tags with capital letters, etc? Is there any front-end protection against having a `foo` and a `Foo` tag, or are they just considered distinct? (I'm not sure which is the better option?)

Reply:
there is no protection against duplicates on the front end. it is added with the unique index now on the backend, which does raise the question of if we wan't to go a step farther and make it case insensitive. i don't have a strong opinion but given the time constraints right now I would rather not make any changes on that.

Reply:
this method is only going to be used by the tag importer for now, since the front end already addresses tags by id there is no reason to change that

Reply:
ok, sounds good, thanks 👍 Just testing my/our assumptions.